src/main.ts

```diff
@@ -11,7 +11,12 @@ async function bootstrap() {
     .setTitle('API')
     .setDescription('Cache Server API documentation')
     .setVersion('1.0')
+    .addBearerAuth(
+      { type: 'http', scheme: 'bearer', bearerFormat: 'JWT' },
+      'access-token',
+    )
     .build();
+
   const document = SwaggerModule.createDocument(app, options);
```

Comment:
Should we be disabling the "try it out" functionality in Swagger as it won't work now that we have auth 🤔 ?

Reply:
On the upper right corner you have an option to auth using jwt token :)

Reply:
Oh, cool, I didn't know Swagger had that. So the steps to use Swagger would be:
1. Generate "IdentityProof" claim
2. Call login CacheServer endpoint (maybe with Swagger) and get back jwt
3. Use jwt in subsequent Swagger calls

How would someone wanting to use Swagger accomplish step 1? Should we give them a script that can use? Or maybe we could provide some endpoint/app/dapp that generates "IdentityProof" claim based on a private key or some connected wallet 🤔 ?

Reply:
Yeah I agree we need to make the step one easier, my first idea was to find a way to run some script on swagger ui load, to log in using metamask and generate token, but I can't find a way to do this using nest swagger module